recipes/pydflow/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pydflow" %}
+{% set version = "1.6.143" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5292c7c1421cdc4055b50f046e2afe7ec771422a05691ec96d8436824ebceba9
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+    - six
+    - python-dateutil
+    - urllib3
+    - certifi
+    - typeguard <3
+    - argo-workflows 5.0.0
+    - jsonpickle
+    - minio
+    - kubernetes
+    - pyyaml
+    - cloudpickle 2.2.0
+    - requests
+    - tqdm
+
+test:
+  imports:
+    - dflow
+    - dflow.python
+
+about:
+  home: https://github.com/deepmodeling/dflow
+  license: LGPL-3.0-only
+  license_family: LGPL
+  license_file: LICENSE
+  summary: Dflow is a Python framework for constructing scientific computing workflows employing Argo Workflows as the workflow engine.
+  doc_url: https://github.com/deepmodeling/dflow
+  dev_url: https://github.com/deepmodeling/dflow
+
+extra:
+  recipe-maintainers:
+    - njzjz
+    - zjgemi
```

Comment:
@zjgemi Please confirm here you are willing to be listed here.

Reply:
I agree to be a maintainer.